site/en/guides/commissioner/build.md

```diff
@@ -0,0 +1,143 @@
+
+## Step 1: Set up OT Commissioner
+
+1.  Clone the OT Commissioner repository:
+    ```
+    $ git clone https://github.com/openthread/ot-commissioner
+    ```
+
+1.  Install dependencies:
+    ```
+    $ cd ot-commissioner
+    $ ./script/bootstrap.sh
+    ```
+
+## Step 2: Build OT Commissioner
+
+OT Commissioner installs to the `/usr/local` directory. If you'd like to change
+your installation directory, set `-DCMAKE_INSTALL_PREFIX`.
+
+> Note: If you change the default installation directory, you'll need to update
+`$PATH` to run `commissioner-cli`. This guide uses an Environment variable to
+work for both default and custom installations.
+
+1.  Build OT Commissioner:
+    ```
+    $ mkdir build
+    $ cd build
+    $ cmake -DCMAKE_INSTALL_PREFIX={/usr/local} -GNinja ..
+    $ ninja -j1
+    ```
+
+1.  Create an Environment variable to run `commissioner-cli` in the next step:
+    ```
+    $ export COMMISSIONER_CLI={/usr/local}/bin/commissioner-cli
+    ```
+
+1.  _Optional_. Run unit tests:
+    ```
+    $ ./tests/commissioner-test
+    ```
+
+## Step 3: Install OT Commissioner
+
+OT Commissioner installs the following to your installation directory:
+
+*   OT Commissioner library and header files
+*   OT Commissioner CLI executable binary
+*   Default configuration files and credentials
+*   Scripts to run OT Commissioner CLI as daemon
+
+```
+$ sudo ninja install
+```
+
+Verify the installation by checking the help menu.
+
+```
+$ $COMMISSIONER_CLI -h
+```
+
+If you installed to the `/usr/local` directory, `commissioner-cli` is available
+from the command line.
+
+```
+$ commissioner-cli -h
+```
+
+## Step 3: Configuration
+
+The OT Commissioner CLI supports both Thread 1.2 Commercial Commissioning Mode
+(CCM) and Thread 1.1 commissioning (Non-CCM). To connect to different Thread
+networks, a JSON configuration file is needed to start OT Commissioner CLI:
+
+*   `ccm-config.json` — The default configuration file for CCM Thread Network.
+*   `non-ccm-config.json` — The default configuration file for Non-CCM Thread
+    Network.
+
+By default, these configuration files are installed in `/usr/local/etc/commissioner`. You can
+also view sample files on the [ot-commissioner GitHub repository](https://github.com/openthread/ot-commissioner/tree/main/src/app/etc/commissioner).
+
+### CCM configuration
+
+To connect to a CCM Thread network, update these fields in `ccm-config.json`:
+
+Field | Description
+----|----
+`DomainName` | Unique identifier within the Enterprise Domain.
+`PrivateKeyFile` | The private key file in PEM format.
+`CertificateFile` | The certificate file in PEM format.
+`TrustAnchorFile` | The trust anchor file in PEM format.
+
+These key and certificate files are used to establish secure sessions between
+the Commissioner and Border Agent.
+
+> Note: If you changed the default installation directory, you'll need to
+update `"ThreadSMRoot" : "/usr/local/etc/commissioner"`.
```

Comment:
@jwhui Who on Eng can confirm this?  It's a new change

Reply:
Hi @jwhui & @Vyrastas I just wanted to follow up. This guide mentions how to install OT Commissioner to a different directory, and the `ThreadSMRoot` points to the default `/usr/local` path - it would need to be updated in the config too?

Reply:
If that's all it's related to, then we're probably good here, just wanted to make @jwhui aware.

Reply:
@wgtdkp , thoughts on above?

Reply:
LGTM, thanks!